arches/app/search/search_export.py

```diff
@@ -50,6 +51,12 @@ def __init__(self, search_request=None):
         search_request.GET["export"] = True
         self.report_link = search_request.GET.get("reportlink", False)
         self.format = search_request.GET.get("format", "tilecsv")
+        self.concept_uuid_format = search_request.GET.get("conceptuuidformat", False)
+        if self.concept_uuid_format:
+            try:
+                self.concept_uuid_format = json.loads(self.concept_uuid_format)
+            except:
+                pass
```

Comment:
It would be better to avoid calling `json.loads` here because it's not actually json that is being loaded. Better to be explicit and do something like:

```
        use_system_value = search_request.GET.get("exportsystemvalue", False)
        self.export_system_value = False if use_system_value == 'false' else use_system_value
```

Reply:
It's not actually json but `json.loads()` still works in translating `'false' -> False` and `'true' -> True`. Explicitly checking the string value means we would have to do:
```
if user_system_value == 'false':
  user_system_value = False
elif user_system_value == 'true':
  user_system_value = True
```
... which feels a bit clunky

Reply:
Agree with @chiatt that passing non-json to json.loads() increases comprehension burden.

What's clunky about this:
```py
new_var = (old_var == "true")
```

Reply:
Alternatively, we have a `str_to_bool()` util for this purpose.

Reply:
fix in latest